# the question that was asked
std_msgs/String question

# the answer to the question
std_msgs/String answer

# the confidence of the answer betweeen [0,1]
float32 score